runtime/src/accounts_db.rs

```diff
@@ -5794,6 +5797,26 @@ impl AccountsDb {
         ancestors: &Ancestors,
         total_lamports: u64,
         test_hash_calculation: bool,
+    ) -> Result<(), BankHashVerificationError> {
+        self.verify_bank_hash_and_lamports_new(
+            slot,
+            ancestors,
+            total_lamports,
+            test_hash_calculation,
+            None,
+            None,
+        )
+    }
+
+    /// Only called from startup or test code.
+    pub fn verify_bank_hash_and_lamports_new(
```

Comment:
Could you add some comments for this `_new` version; what it will be used work, or maybe better, _when_ it will be used? I assume the non-`_new` version will also be removed later too?

Reply:
I'll clean this up as we go.